changelog.md

```diff
@@ -5,6 +5,9 @@
 **Added:**
 - bpk-theming:
   - New web theming utility, see https://backpack.github.io/components/utilities/theming
+- bpk-component-theme-switcher:
+  - New component to help switching themes on the fly
```

Comment:
Is this component public?

Reply:
I guess so, RTL is. should it not be public?